cache/cache.go

```diff
@@ -68,13 +70,13 @@ func RebuildCmd(c Cache, src, dst string) (err error) {
 	src = filepath.Clean(src)
 	src, err = filepath.Abs(src)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "absolute path failed for %s", src)
```

Comment:
You have added this dependency even if we aren't using it in any plugin. Until now we always used only fmt and logrus.